tools/autograd/gen_variable_type.py

```diff
@@ -668,6 +720,73 @@ def emit_check_inplace() -> List[str]:
             return []
         return [f'check_inplace({arg.name}, _any_requires_grad);' for arg in differentiable_outputs]
 
+    def emit_fw_derivatives() -> List[str]:
+        content: List[str] = []
+        for derivative in fw_derivatives:
+            res = derivative.var_name
+            if f.func.name.name.inplace:
+                # TODO update this when inplace namings are unified
```

Comment:
Link to issue? Also I'm not sure what "inplace namings are unified" refers to

Reply:
It corresponds to this comment: https://github.com/pytorch/pytorch/blob/0b8bd2261450d21e0dd0ae259aa06994d1a0d9b8/tools/codegen/api/cpp.py#L167-L170
Basically the "self" is hardcode throughout the code-base and we could do better.